README.md

```diff
@@ -123,7 +123,7 @@ Properties of `hnswlib.Index` that support reading and writing:
         
   
 #### Python bindings examples
-[See more examples here](examples/EXAMPLES.md)
+[See more examples here](examples/python/EXAMPLES.md)
```

Comment:
Maybe give a one-two word description of the examples

Reply:
done